src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -1,31 +1,97 @@
 package com.endava.internship.mocking.service;
 
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.User;
+import com.endava.internship.mocking.repository.InMemPaymentRepository;
+import com.endava.internship.mocking.repository.InMemUserRepository;
+import com.endava.internship.mocking.repository.PaymentRepository;
+import com.endava.internship.mocking.repository.UserRepository;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.*;
+
 @ExtendWith(MockitoExtension.class)
 class PaymentServiceTest {
 
+    private PaymentService underTest;
+    private UserRepository userRepository;
+    private PaymentRepository paymentRepository;
+    private ValidationService validationService;
 
     @BeforeEach
     void setUp() {
-
+        userRepository = spy(InMemUserRepository.class);
+        paymentRepository = spy(InMemPaymentRepository.class);
+        validationService = mock(BasicValidationService.class);
+        underTest = new PaymentService(userRepository, paymentRepository, validationService);
     }
 
     @Test
     void createPayment() {
+        ArgumentCaptor<Payment> paymentCaptor = ArgumentCaptor.forClass(Payment.class);
+        Integer userId = 1;
+        Double amount = 44.44;
```

Comment:
Recommend to declare these as class constants, and use them whenever you need in tests